website/docs/r/automation_schedule.html.markdown

```diff
@@ -0,0 +1,73 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_automation_schedule"
+sidebar_current: "docs-azurerm-resource-automation-schedule"
+description: |-
+  Creates a new Automation Schedule.
+---
+
+# azurerm\_automation\_schedule
+
+Creates a new Automation Schedule.
+
+## Example Usage
+
+```
+resource "azurerm_resource_group" "example" {
+ name = "resourceGroup1"
+ location = "West Europe"
+}
+
+resource "azurerm_automation_account" "example" {
+  name                = "account1"
+  location            = "${azurerm_resource_group.example.location}"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+  sku {
+        name = "Free"
+  }
+}
+
+resource "azurerm_automation_schedule" "example" {
+  name                = "schedule1"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+  account_name        = "${azurerm_automation_account.example.name}"
+  frequency           = "OneTime"
+  timezone            = "Central Europe Standard Time"
+  start_time	      = "2014-04-15T18:00:15-07:00"
```

Comment:
can we update the timezone on this to be `+02:00` - otherwise users seeing this will get a perpetual diff:

```
  ~ azurerm_automation_schedule.example
      start_time: "2017-10-16T03:00:00+02:00" => "2017-10-15T18:00:15-07:00"
```

Reply:
also using this example it appears that seconds aren't a valid interval here - do we need to validate for that?

```
  ~ azurerm_automation_schedule.example
      start_time: "2017-10-15T18:00:00+02:00" => "2017-10-15T18:00:15+02:00"
```

Reply:
I think seconds are valid in the start time. You can set the start time on the Azure Website in second accuracy so i think 18:00:05 is a valid start time for a schedule.

Reply:
Done. I only modified the timezone in the start_time.